demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -78,12 +78,14 @@ def return_release_notes_path(self, input_version: str):
 
     @staticmethod
     def get_display_name(file_path):
-        struct = StructureValidator(file_path=file_path)
+        struct = StructureValidator(file_path=file_path, is_new_file=True)
         file_data = struct.load_data_from_file()
         if 'name' in file_data:
             name = file_data.get('name', None)
         elif 'TypeName' in file_data:
             name = file_data.get('TypeName', None)
+        elif 'brandName' in file_data:
+            name = file_data.get('TypeName', None)
```

Comment:
Guessing it should be:
```suggestion
            name = file_data.get('brandName', None)
```